docs/modules/default-architecture/pages/storing.adoc

```diff
@@ -2,13 +2,13 @@
 
 We recommend LangChain's OpenAIEmbeddings class for storing your embeddings in a vector store.
 
-We recommend DataStax {db-vector} to store your embeddings. It's a managed service that uses Cassandra under the hood, and is a good choice if you're already using Cassandra or want to use Cassandra for other parts of your application. {db-vector} integrates with LangChain as a vector store using the https://python.langchain.com/docs/integrations/vectorstores/astradb[AstraPy client].
+We recommend {company} {db-vector} to store your embeddings. It's a managed service that uses Cassandra under the hood, and is a good choice if you're already using Cassandra or want to use Cassandra for other parts of your application. {db-vector} integrates with LangChain as a vector store using the https://python.langchain.com/docs/integrations/vectorstores/astradb[AstraPy client].
 
 include::examples:partial$prerequisites.adoc[]
 
-== Store embeddings in Astra vector database
+== Store embeddings in {db-vector} database
 
-This code embeds the loaded `Documents` from the xref:splitting.adoc[] example and stores the embeddings in the AstraDB vector store.
+This code embeds the loaded `Documents` from the xref:splitting.adoc[] example and stores the embeddings in the Astra DB vector store.
```

Comment:
Should we specify Astra DB Serverless here?

Reply:
Yes.